prime-router/build.gradle.kts

```diff
@@ -48,6 +48,16 @@ plugins {
     id("io.swagger.core.v3.swagger-gradle-plugin") version "2.2.22"
 }
 
+// retrieve the current commit hash
+val commitId by lazy {
+    val stdout = ByteArrayOutputStream()
+    exec {
+        commandLine("git", "rev-parse", "--short", "HEAD")
+        standardOutput = stdout
+    }
+    stdout.toString(StandardCharsets.UTF_8).trim()
+}
+
```

Comment:
Does this end up running in CI? Are we sure that all the contexts this runs in has git?

As an alternative to running git, any thoughts on reading `.git/HEAD` and `.git/refs/<current-HEAD`?

Reply:
Nah if we know it works in all environments (which is obvious if already in master) then I'm good 👍.